brainiak/reprsimil/brsa.py

```diff
@@ -686,7 +688,7 @@ def _fit_RSA_UV(self, X, Y, X0,
         n_C = np.size(X, axis=1)
         l_idx = np.tril_indices(n_C)
 
-        np.random.seed(self.rand_seed)
+        self.random_state_ = check_random_state(self.random_state)
```

Comment:
Please document this attribute in the class docstring. Could be something like this:
```
random_state_: `RandomState`
    Random number generator initialized using random_state
```